package.json

```
@@ -32,7 +40,7 @@
     "test": "run-s \"test:*\"",
     "test:lint": "lerna exec --stream -- npx eslint .",
     "test:unit": "lerna run --stream test:unit",
-    "test:e2e": "lerna run --stream test:e2e",
+    "test:e2e": "lerna run --stream --concurrency 1 test:e2e",
```

probably don't want to run e2e tests in parallel, (a) port clashes unless we use get-port, and (b) figuring out what happened in the log is probably impossible.

---

If you really don't want to run stuff in parallel, I think all you need to do is this:

```suggestion
    "test:e2e": "lerna run test:e2e",
```

`--stream` will prefix each log line w/ the package name, but I don't know if that only works with `--parallel`, and also don't know how that interacts with `--no-stream`.

I'm certain the port issue can be resolved; I'd say if getting things running in parallel in a sane manner can be accomplished quickly, might as well go for it. it's likely to improve performance--especially in a local env.

---

without `--stream` you can't see log output until the whole process completes, which sucks.